project/.github/workflows/ci.yml

```
@@ -0,0 +1,47 @@
+name: CI
+
+on: [push, pull_request]
+#on:
+#  push:
+#    branches: [master]
+#  pull_request:
```

I think this trigger syntax is wrong: `push` and `pull_request` are objects, so you should probably use the commented part
```suggestion
on:
  push:
    branches: [master]
  pull_request:
```

---

Figured out the issue. I had this file in the wrong directory which I fixed below 😅 
[project/.github/workflows/ci.yml → .github/workflows/ci.yml](https://github.com/Alkaar/resy-booking-bot/pull/38/commits/eed725d16c355310a49f009b3493486d0e2e3842#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f)